### DISCUSSION TITLE
Deploy Magma with Vagrant

### DISCUSSION THREAD

**Discussion by carstair** [magma/magma#15191] [Q&A] [answered]
I have a Question, i tried to install magma with vagrant, then when i try to deploy in the enviroment this message come: ==> magma: You assigned a static IP ending in ".1" to this machine.
==> magma: This is very often used by the router and can cause the
==> magma: network to not work properly. If the network doesn't work
==> magma: properly, try changing this IP.
There are errors in the configuration of this machine. Please fix
the following errors and try again:
Vagrant:
* Unknown configuration section 'vbguest'.
How can i fix that?, where is the file to modify to change the "magma box"  and change the IP address?

![image](https://user-images.githubusercontent.com/36524233/236698396-14fba76f-3082-4953-b05c-5e1021c6ed65.png)

**Comment by panyogesh** [comment, marked as answer]
This is because of a missing plugin in Vbox : "vagrant plugin install vagrant-vbguest". This is not directly related to Magma.

**Comment by panyogesh** [comment]
Closing the older ticket.
